decidim-api/spec/controllers/queries_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+module Decidim
+  module Api
+    describe QueriesController, type: :controller do
+      routes { Decidim::Api::Engine.routes }
+      let!(:participatory_process) { create(:participatory_process) }
+      let!(:other_participatory_process) { create(:participatory_process) }
+
+      before do
+        @request.env["decidim.current_organization"] = participatory_process.organization
+      end
+
+      it "executes a query" do
+        post :create, params: { query: "{ processes { id }}" }
```

Comment:
No need, just checking the schema is being executed. The rest of it is tested in the types.